http-common/src/proxy.rs

```diff
@@ -143,7 +143,40 @@ fn identity_to_tls_connector(
 
     let cert_store = tls_connector.cert_store_mut();
     for trusted_cert in trusted_certs {
-        cert_store.add_cert(trusted_cert.clone())?;
+        if let Err(err) = cert_store.add_cert(trusted_cert.clone()) {
```

Comment:
You are getting rid of this code because you don't expect to support openssl 1.0?

Reply:
I'm adding code because we support openssl 1.0

Reply:
Geez - I shouldn't read reviews late on aFriday night apparently.